assets/dev/js/editor/controls/repeater-row.js

```diff
@@ -73,7 +73,12 @@ module.exports = Marionette.CompositeView.extend( {
 		let title = '';
 
 		if ( titleField ) {
-			title = Marionette.TemplateCache.prototype.compileTemplate( titleField )( this.model.parseDynamicSettings() );
+			title = Marionette.TemplateCache.prototype.compileTemplate( titleField )(
+				{
+					...this.model.parseDynamicSettings(),
+					itemIndex: this.getOption( 'itemIndex' ),
```

Comment:
@louiswol94 I added this so that the tag was available in `title_field` and could be used. Do you think that there is a better way of achieveing this?

Reply:
@nicoladj77 I do not understand, can you please elaborate a bit more? Tnx.

Reply:
@louiswol94 so I am using this here https://github.com/elementor/elementor/pull/27322/files#diff-405e5ccb597bae3aa605f6db5467dffc4a03e6ab37fccff7b27cd2d4e39451beR256

in the mocukp we have to display "Image #1" instead of "Item #1" which is the default title. I checked the repeater code and it uses that `itemIndex` property which is not exposed for parsing the `title_field` template. Or at least I didn't see it in parseDynamicSettings() so I exposed it.

Reply:
@louiswol94 rolled back changes